stock_orderpoint_mrp_link/tests/test_stock_orderpoint_mrp_link.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
@hveficent Mixing LGPL and AGPL licenses. Could you clarify ?

Reply:
It should have been LGPL, good catch :smile: